change/@fluentui-react-charting-effaa597-2236-4e8c-b8f3-91a2422c9f44.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Optimizing code for perfromance improvement",
+  "packageName": "@fluentui/react-charting",
+  "email": "ankityadav@microsoft.com",
```

Comment:
Update the PR description with the react app based baseline and improvement.

Reply:
Done